.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,184 @@
+
+      - name: Stop and remove the benchmark container
+        run: docker stop my_container && docker rm my_container
+
+      - name: Query metadata get processed height
+        id: query_meta
+        env:
+          PGPASSWORD: postgres
+        run: |
+          chmod +x .github/workflows/scripts/benchmark/queryMeta.sh
+          .github/workflows/scripts/benchmark/queryMeta.sh
+
+      - name: Pre-save running specs
+        run: |
+          echo "benchmark-time=${{github.event.inputs.benchmark-time}}" > output/benchmark/benchmark.log
+          echo "indexer=${{steps.query_meta.outputs.runner_node}}" >> output/benchmark/benchmark.log
+          echo "indexerVersion=${{steps.query_meta.outputs.indexer_version}}" >> output/benchmark/benchmark.log
+          echo "deployment=${{github.event.inputs.deployment}}" >> output/benchmark/benchmark.log
+          echo "network-endpoint=${{github.event.inputs.network-endpoint}}" >> output/benchmark/benchmark.log
+          echo "disableHistorical=${{github.event.inputs.disableHistorical}}" >> output/benchmark/benchmark.log
+          echo "batch-size=${{github.event.inputs.batch-size}}" >> output/benchmark/benchmark.log
+          echo "workers=${{github.event.inputs.workers}}" >> output/benchmark/benchmark.log
+          if [ -n "${{github.event.inputs.others}}" ]; then
+            echo "others=${{github.event.inputs.others}}" >> output/benchmark/benchmark.log
+          fi
+          echo "Start Height: ${{steps.query_meta.outputs.start_height}}" >> output/benchmark/benchmark.log
+          echo "Last Processed Height: ${{steps.query_meta.outputs.last_processed_height}}" >> output/benchmark/benchmark.log
+          echo "------------------------------------" >> output/benchmark/benchmark.log
+
+      - name: Prepare benchmark log
+        run: |
+          if grep -qi "benchmark" output/benchmark/indexing.log; then
+            grep -i "benchmark" output/benchmark/indexing.log >> output/benchmark/benchmark.log
+            cat output/benchmark/benchmark.log
+          else
+            echo "No lines containing 'benchmark' found in output/benchmark/indexing.log" >> output/benchmark/benchmark.log
+          fi
+
+      - name: Get current time
+        uses: josStorer/get-current-time@v2
+        id: current-time
+        with:
+          format: YYYYMMDD-HHmm
+          utcOffset: "+12:00"
+
+      - name: Upload reports
+        uses: actions/upload-artifact@v3
+        with:
+          name: benchmark-${{steps.current-time.outputs.formattedTime}}-${{steps.query_meta.outputs.indexer_version}}
+          path: output/benchmark/
+
+
+      - name: Post to a Slack channel
+        id: slack
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+          # Slack channel id, channel name, or user id to post message.
+          # See also: https://api.slack.com/methods/chat.postMessage#channels
+          channel-id: 'C02156BV42V'
```

Comment:
Can you comment the name of this channel please. I cant remember what one to look in slack.

Reply:
It is a App(bot) we can invite to all channel